docs/source/migration.mdx

```diff
@@ -146,15 +157,27 @@ import typeDefs from "./typeDefs";
 
   const server = new ApolloServer({
     schema,
+    context() {
+      // lookup userId by token, etc.
```

Comment:
I wonder if it's obvious enough where userId comes from from this comment. It might be a good idea to show it "for real", so you can see that context and onConnect take different parameters...